### ISSUE TITLE
Form input placeholder value is not escaped

### ISSUE THREAD

**Issue by barrystaes** [contao/core#8505]
In Contao v3.5.16 when setting a placeholder value for a Security Question input, i noticed that it wasnt escaped properly. This results in mangled HTML or injections even.

I entered this as placeholder value: `Uw antwoord om te bevestigen dat u geen "spam bot" bent`

And saw this in the HTML for visitors:
```
<input type="text" name="c1715a89214715466a3d1921ffa47cd05" id="ctrl_19" class="captcha mandatory" value="" placeholder="Uw antwoord om te bevestigen dat u geen "spam bot" bent" maxlength="2" required>
```
The same problem **may exist for other form types, i have not tested further**.

**Comment by barrystaes** [comment]
Mildly interesting, but i noticed that Chrome did interpret this as such:
```
<input type="text" name="c91b3dff314d57fd6ec0e8886d4351b51" id="ctrl_19" class="captcha mandatory" value="" placeholder="Uw antwoord om te bevestigen dat u geen " spam="" bot"="" bent"="" maxlength="2" required="">
                                                                                                                                                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by leofeyer** [comment]
Fixed in 49d2addff30a92a0d47835ef472b2ded06c110b0.
